### DISCUSSION TITLE
Track the longest #co-working-room session

### DISCUSSION THREAD

**Discussion by ebanner** [Virtual-Coffee/virtualcoffee.io#1155] [New Ideas]
I saw this co-working room today

![image](https://github.com/Virtual-Coffee/virtualcoffee.io/assets/2068912/1513be3b-ca65-4678-b74f-b6c5fa4b3620)

That's a lot of replies! Seems like a pretty baller co-working session

Might be fun to track whatever was the longest continuous active co-working room session, like the record for longest active co-working session

**Comment by meg-gutshall** [comment]
I like it! I'm pretty sure I hold the record though... 👑
